cypress/integration/team-members/list.js

```diff
@@ -0,0 +1,36 @@
+describe('List team members', () => {
+  before(() => {
+      cy.task('db:seedTeamMembers')
+    }
+  )
+  it('Should not have permissions when not logged in', () => {
+    cy.visit('http://localhost:3000/team-members/list')
+
+    cy.contains('Welcome to SalesWell')
+  })
+  
+  it('Should have permissions when logged in as user', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/team-members/list')
+
+    cy.contains('admin user').should('exist')
+    cy.contains('admin@example.com').should('exist')
+    cy.contains('ADMIN').should('exist')
+
+    // This is the test for team members with ADMIN role
+    for (let i = 10; i >= 6; i--) {
```

Comment:
Why are you decreasing from 10 to 6 and not going up right from 6 to 10? Alos, let's check the role on  the card too.